coral/src/domain/topic/topic-api.ts

```diff
@@ -14,10 +16,15 @@ const getTopics = async ({
 }: {
   currentPage: number;
   environment: Environment;
-  teamName?: string;
+  teamName: Team | null;
   searchTerm?: string;
 }): Promise<TopicApiResponse> => {
-  const team = teamName && teamName !== "All teams" ? teamName : null;
+  // "ALL_TEAMS_VALUE" represents topic list without
+  // the optional team parameter
+  // where we still need a way to represent an
+  // option for "Select all teams" to users
+  const team = teamName && teamName !== ALL_TEAMS_VALUE && teamName;
```

Comment:
This can be simplified. I would do something like `isString(teamName) && teamName !== ALL_TEAMS_VALUE`. I like human readable type guard as my head is not good at computing booleans.

Reply:
sounds good!